### PR TITLE
Add files to interpret and implement branch.json

### PR DESCRIPTION
**Pull request by adrianyg7**
Add files to interpret and implement branch.json

### CODE REVIEW

File: Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj
Comment (jdee) [inline Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj:1215]: We should probably back this out, at least manually (with vi or whatever). I can do that if it's tough otherwise.

**Review by jdee — COMMENTED** [review]
Looks good. :+1: I guess we could stand to have some unit tests for the code I wrote and never bothered writing tests for. But I think this is the right idea. I'll leave it to Ernest to approve the PR.

File: Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj
Comment (adrianyg7) [reply Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj:1215]: Right, could you please do that?
I had to use our team and even when reverting outside xcode seems easy, I wouldn't like to mess with it.

Comment (jdee) [reply Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj:1215]: Ernest or I can revert once this is merged, since this is from your fork. You can also use any text editor to modify the field in the file. You should still be able to build and deploy to a simulator. You can also perhaps go to a previous commit from before this change, copy out `Branch-TestBed.xcodeproj/project.pbxproj` to `/tmp` or anywhere, then copy it back in after reverting to this `HEAD` revision, depending if there were other changes to the file in the same commit. Not a big deal. There are numerous ways to deal with it now or later.

**Review by echo-branch — APPROVED** [review]
Looks good.  I'll cleanup the project settings on the staging branch.

Update.  Actually fixed it with the App Clip PR.

**Pull request merged**
